fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PlanTranslatorContext.java

```diff
@@ -75,6 +76,11 @@ public class PlanTranslatorContext {
      */
     private final Map<SlotId, ExprId> slotIdToExprId = Maps.newHashMap();
 
+    /**
+     * index from Nereids' slot to legacy slot.
+     */
```

Comment:
The comments need to be clearer to explain it is used for lambda function

Reply:
done